backend/test/users_test.js

```diff
@@ -734,3 +737,101 @@ describe("Test get student completed courses", () => {
   });
 });
 
+describe("Testing Login", () => {
+  before(async () => {
+    await users.insertStudentUser(
+      "team",
+      "primus1234",
+      "testing@gmail.com",
+      250502459
+    );
+  });
+
+  it(`Empty username responds with ${expectedUsernameNotEmpty}`, function(done) {
+    users.login(null, "primus1234").catch(error => {
+      return new Promise(function(resolve) {
+        assert.equal(error.message, expectedUsernameNotEmpty);
+        resolve();
+      }).then(done);
+    });
+  });
+
+  it(`Empty password responds with ${expectedPasswordNotEmpty}`, function(done) {
+    users.login("team", null).catch(error => {
+      return new Promise(function(resolve) {
+        assert.equal(error.message, expectedPasswordNotEmpty);
+        resolve();
+      }).then(done);
+    });
+  });
+
+  it(`Empty fields responds with ${expectedPasswordNotEmpty}`, function(done) {
```

Comment:
Did you mean responds with ${expectedUsernameNotEmpty} ?

Reply:
thanks!